src/iam/iam-base.ts

```diff
@@ -25,7 +25,7 @@ import { ClaimManager } from "../../ethers/ClaimManager";
 import { JWT } from "@ew-did-registry/jwt";
 import { ICacheServerClient } from "../cacheServerClient/ICacheServerClient";
 import { detectExecutionEnvironment, ExecutionEnvironment } from "../utils/detectEnvironment";
-import { connect, NatsConnection, JSONCodec, Codec } from "nats.ws";
+import { connect, NatsConnection, Codec, JSONCodec } from "../../node_modules/nats.ws/lib/src/mod.js";
```

Comment:
Are `Codec` and `JSONCodec` not importable from `nats.ws` index?

Reply:
"The transpilation process seems to not pick up on the right library for nats.ws. To work around the issue, simple insure you import from the cjs file directly" - I confirmed that it also referes to JSONCodes and Codec